solr_schema_init/create_isb_core_schema.py

```diff
@@ -225,5 +225,17 @@ def createCollection():
 
 createField("producedBy_samplingSite_location_h3", "string", True, True, None)
 createField("producedBy_samplingSite_location_cesium_height", "pfloat", True, True, None)
+# Nested document support
+# Note that the solr docs indicate we need these fields, but they already existed in our schema, keeping here for
+# reference purposes
+# createField("_root_", "string", False, True)
+# addFieldType({
+#     "name": "_nest_path_",
+#     "class": "solr.NestPathField",
+# })
+# createField("_nest_path_", "nest_path", True, True)
```

Comment:
These 3 are the new fields that each record in the schema could potentially have.